presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/ThriftConnectorIndex.java

```diff
@@ -29,6 +30,7 @@
         implements ConnectorIndex
 {
     private final DriftClient<PrestoThriftService> client;
+    private final Map<String, String> thriftHeader;
```

Comment:
Nit: this should be `thriftHeaders` everywhere (plural since there are multiple headers)